packages/react-native/src/primitives/Icon/types.ts

```diff
@@ -1,5 +1,16 @@
 import { ColorValue, ImageProps, ImageStyle } from 'react-native';
 
+export type Size = 'xs' | 'small' | 'medium' | 'large' | 'xl';
+export type IconSizes = Record<Size, number>;
+
+export const iconSizes: IconSizes = {
```

Comment:
Can we put this in a _constants.ts_ file rather than including it with the types?

Reply:
sure